dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -230,6 +230,11 @@ DisableAutoBuildLabel=Disable &Auto Build
 EnableDisableAutoBuildJob=Updating auto build setting for: {0}
 ErrorOnEnableDisableAutoBuild=An error occurred while enabling or disabling auto build for application: {0}.
 
+EnableInjectMetricsLabel=Enable &Inject Metrics
```

Comment:
I is already in use as a hot key for Import project

Reply:
Changed to `Enable inject me&trics`